h5c/vic/src/main/locale/en_US/com_vmware_vic.properties

```diff
@@ -196,8 +190,23 @@ vch.registry.additionalRegistryCertificates=Additional registry certificates
 vch.registry.addRegCertsSubText=Add registry certificates for those registries which need custom or self-signed certificates, like the vSphere Integrated Containers Registry.
 vch.registry.selectX508CertPemFile=Select X.509 certificate pem file
 vch.registry.expires=expires
-vch.registry.replace=Replace
-vch.registry.select=Select
+
+# vch operations
+vch.operations.title=Operations User
+vch.operations.provideInfo=Provide vSphere user credentials for the VCH to use when interacting with vSphere
+vch.operations.specifyInfo=Specify a non-administrative account to limit permissions for day-to-day VCH operations.
```

Comment:
It looks like one Info line (196) has no period, and next line (197) has one. Let's make sure it is consistent.